.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
+max_line_length = 120
+tab_width = 4
```

Comment:
is it okay to make it 2 instead?
4 still takes up quite some screen real estate for me.

same as the `indent_size` on top.

Reply:
I like 2 as well personally, but defaulted to the standard Kotlin coding conventions so it was typical to other Kotlin code: https://kotlinlang.org/docs/coding-conventions.html#indentation